src/modules/users/infra/http/controllers/ForgotPasswordController.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response } from 'express'
+import { container } from 'tsyringe'
+import SendForgotPasswordEmailService from '@modules/users/services/SendForgotPasswordEmailService'
+
+export default class ForgotPasswordController {
+  public async create(request: Request, response: Response): Promise<Response> {
+    const { email } = request.body
+
+    const sendForgotPasswordEmail = container.resolve(
+      SendForgotPasswordEmailService
+    )
+
+    await sendForgotPasswordEmail.execute({
+      email
+    })
+
+    return response.status(204).json()
```

Comment:
Quando a response não precisa de um retorno com dados, basta usar o [`sendStatus`](https://expressjs.com/pt-br/api.html#res.sendStatus)

```suggestion
    return response.sendStatus(204)
```